src/hypertable.c

```diff
@@ -1017,14 +1017,19 @@ hypertable_validate_constraints(Oid relid)
  *   (you can work around this, but is messy). This issue, b.t.w., broke one
  *   of the tests.
  *
- * - A trigger, especially an "internal" one, is transparent (doesn't show up
+ * - An internal trigger is transparent (doesn't show up
  *	 on \d+ <table>) and is automatically removed when the extension is
  *	 dropped (since it is part of the extension). Internal triggers aren't
  *	 inherited by chunks either, so we need no special handling to _not_
- *	 inherit the blocking trigger.
+ *	 inherit the blocking trigger. However, internal triggers are not exported
+ *   via pg_dump. Because a critical use case for this trigger is to ensure
```

Comment:
indentation here doesn't match the line above. `pgindent`?

Reply:
done